runtime/src/iree-amd-aie/CMakeLists.txt

```diff
@@ -13,3 +13,5 @@ endif()
 # (.vmfb file), and so the schema sub-directory is required even when not
 # building the XRT driver code.
 add_subdirectory(schemas)
+
+add_subdirectory(runtime)
```

Comment:
can we find a different name than runtime. I think an uninitiated reader would get confused looking for the iree hal runtime in this folder. Also comment on top mentioning what's in this directory and what it does.